### PR TITLE
storage_engine removed since 5.7.5 replaced by default_storage_engine…

### PR DESCRIPTION
**Pull request by evertsemeijn**
… and small typo in numbering sections

### CODE REVIEW

File: doc/manual/installation.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ Create a user for Huginn do not type the `mysql>`, this is part of the prompt. C
 
 Ensure you can use the InnoDB engine which is necessary to support long indexes
 
-    mysql> SET storage_engine=INNODB;
+    mysql> SET default_storage_engine=INNODB;
```
Comment (dsander) [inline doc/manual/installation.md]: Does this also work for older versions? I think we should at least support 5.5 and up.

**Comment by evertsemeijn** [comment]
I couldn't set innodb with the previous command on latest MySQL. So than default_storage_engine has to be added as an option for ≥5.7.5

**Comment by dsander** [comment]
We should be good for mysql >= 5.5 which was released six years ago, so I would say we do not need to support 5.4 and below?

File: doc/manual/installation.md (hunk 1 above)
Comment (cantino) [inline doc/manual/installation.md]: `default_storage_engine` appears to exist in the 5.5 docs, so I guess this is fine?

Comment (dsander) [inline doc/manual/installation.md]: I agree

**Comment by dsander** [comment]
Thank you @evertsemeijn!

**Pull request merged**